environment.go

```diff
@@ -371,6 +371,7 @@ func (e *envState) initDB(ctx context.Context, settings DBSettings, tracer trace
 
 	opts := options.Client().ApplyURI(settings.Url).SetWriteConcern(settings.WriteConcernSettings.Resolve()).
 		SetReadConcern(settings.ReadConcernSettings.Resolve()).
+		SetTimeout(5 * time.Minute).
```

Comment:
Can you share a bit more about how you reached this conclusion? 

>  Looking at DB activity for the cluster it seems like 5 minutes is a good conservative limit, but I'm fine to tweak this if there are concerns about any workflows that might run a query that long.

Are there any queries that take longer that seem like they may be an edge case but legitimate? Or does it seem like most that take more than x amount of time are running until they time out?

Reply:
That makes sense. The ones that did exceed 5 minutes, by how much did they exceed? If they went to like 10 or 15 minutes, it might make sense to up it a bit more. If it was a lot more, I think 5 minutes makes sense.

Reply:
The ones that exceeded 5 minutes were in the 5-15 minute range but since they weren't originating from Evergreen,  this change won't affect them.